.script/tests/asimParsersTest/runAsimTestersOnForkedRepo.ps1

```diff
@@ -0,0 +1,97 @@
+Param([string]$fork, [string]$branch, [string]$repoBaseFolder)
+
+$global:failed=0
+$global:subscriptionId="419581d6-4853-49bd-83b6-d94bb8a77887"
```

Comment:
Why? as far as I know subscription id is not a secret.
How can we know for sure?

Reply:
It is not a secret, but it is identifying information, which has both privacy issues, as well as security issues. In any case, I once had an escalation from a partner because a review found (a year after the fact) that I had a demo in which their subscription id popped up on the screen when I selected a scope... I also had a Microsoft GitHub review that asked me to remove a workspace ID, which is similar.